### PR TITLE
Cv 495 cohort deletion

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <dt class="govuk-summary-list__key">
+                        Number of apprentices
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.DraftApprenticeshipsCount
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Apprentices
```
Comment (chrisfoster76) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml]: According to the screen designs, this should say `Apprenticeships`

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                            <text>@course.NumberOfDraftApprenticeships x @course.DisplayCourseName</text><br />
+                        }
+                    </dd>
+                </div>
+            </dl>
+            <div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.ConfirmDeletion, "govuk-form-group--error")">
+                <fieldset class="govuk-fieldset">
+                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                        <h2 class="govuk-fieldset__heading govuk-!-margin-bottom-3">
+                            Are you sure that you want to delete all apprentice details?
```
Comment (chrisfoster76) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml]: According to the screen designs, there should be no `that` in this text

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml (hunk 1 above)
Comment (Pauljgraham) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml]: good spot

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml (hunk 2 above)
Comment (Pauljgraham) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/ConfirmDelete.cshtml]: I blame James King

**Pull request merged**